.github/workflows/config.yml

```diff
@@ -0,0 +1,27 @@
+name: run-trailmap-tests
+on: push
+jobs: 
+  test_py3:
+    runs-on: ubuntu-latest
+    container:
+      image: cyclus/cycamore:latest
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
```

Comment:
The image you are loading with `cyclus/cycamore` expects a particular version of Python, so we can leave out this matrix as well as the setup python action.

Reply:
@nuclearkatie - am I right about Python versions?